src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
-body {
+@import 'var';
+
+* {
   margin: 0;
+  box-sizing: border-box;
 }
+
+.calendar {
+  &-block {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1px;
+    width: $sheet-size * 7 + 10px;
+    margin-left: 10px;
+
+    @each $positionDay, $position in $calendar {
+      &__day--#{$positionDay} :nth-child(1) {
+        margin-left: $position;
+      }
+    }
+
+    @each $finalDay in $end-day {
+      @for $day from 1 through $finalDay {
+        &__month--#{$finalDay} :nth-child(#{$day}) {
+          border: 1px solid #000;
+          background-color: #fff;
+          text-align: center;
+          line-height: 100px;
+          font-size: 30px;
+          font-family: Arial, sans-serif;
```

Comment:
it's better to apply common styles for the basic class instead of generating the same code for each class with a modifier